New Checkout Drop-in v1.4.0-beta2 (#273)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/payment-methods.mdx

import OptionsTable from '@components/OptionsTable.astro';

> The `PaymentMethods` container is designed to manage and display the available payment methods during the checkout process. You can configure it to decide whether the container should not set the payment method automatically when it is selected, display or not a payment method, show an icon beside of the label, display or not the label and finally provide a specific handler to render the payment method.

Reviewer: ```suggestion
Use the `PaymentMethods` container to manage and display the available payment methods during the checkout process. Configuration options:
- Set the payment method automatically or manually (starting without a selected payment method)
- Show an icon beside of the label
- Display or hide the label
- Provide a specific handler to render the payment method
```

}
```

> <li>The `displayTitle (*)` property is inherited from `TitleProps` interface; this is used to decide whether the title is displayed or not.</li>

Reviewer 2: ```suggestion
<li>The `displayTitle (*)` property is inherited from the `TitleProps` interface. It is used to determine whether to display the title.</li>
```

> <li>The `setOnChange` property has been _deprecated_ and will be removed in future versions. It has been replaced by the following one: `slots.Methods.<payment-method-code>.setOnChange`.</li>

Reviewer 2: ```suggestion
<li>The `setOnChange` property has been _deprecated_ and will be removed in future versions. It has been replaced by `slots.Methods.<payment-method-code>.setOnChange`.</li>
```

> <li>The `UIComponentType` property is a string containing the name of the UI component type to be used as a selector for each payment method. The UI components available are: 'ToggleButton' or 'RadioButton'.</li>

Reviewer 2: ```suggestion
<li>The `UIComponentType` property is a string containing the name of the UI component type to be used as a selector for each payment method. The available UI components are: `ToggleButton` or `RadioButton`.</li>
```

<li>The `slots` property is an object containing the following properties:</li>

#### Title property (*)

> The `Title` property is inherited as a result of merging the `slots` from `LoginFormProps` and `TitleProps`; this is a handler used to render a custom title content.

Reviewer 2: ```suggestion
The `Title` property is inherited as a result of merging the `slots` from `LoginFormProps` and `TitleProps`. This is a handler that renders custom title content.
```
